crates/experimentation_platform/src/api/experiments/handlers.rs

```diff
@@ -140,10 +141,8 @@ async fn create(
         );
     }
 
-    // Checking if context is a key-value pair map
-    if !req.context.is_object() {
-        return Err(bad_argument!("Context should be map of key value pairs."));
-    }
+    // validating context
+    validate_context(&req.context)?;
```

Comment:
Shouldn't this be in context create endpoint?

Reply:
No really, validate_context, specifically checks whether the user has passed `variantIds` dimensions in the payload or not, which shouldn't be restricted on CAC side.

Reply:
@knutties shouldn't variantIds dimension should be reserved to be put in context via experimentation only 
and context apis in cac should validate this ?

Reply:
only via experimentation.  It is an experimentation filter.  CAC should not treat any dimension as special.